_vendor/github.com/moby/buildkit/frontend/dockerfile/docs/rules/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Build checks
+description: |
+  BuildKit has built-in support for analyzing your build configuration based on
+  a set of pre-defined rules for enforcing Dockerfile and building best
+  practices.
+keywords: buildkit, linting, dockerfile, frontend, rules
+---
+
+BuildKit has built-in support for analyzing your build configuration based on a
+set of pre-defined rules for enforcing Dockerfile and building best practices.
+Adhering to these rules helps avoid errors and ensures good readability of your
+Dockerfile.
+
+Checks run as a build invocation, but instead of producing a build output, it
+performs a series of checks to validate that your build doesn't violate any of
+the rules. To run a check, use the `--check` flag:
+
+```console
+$ docker build --check .
+```
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="./stage-name-casing.md">StageNameCasing</a></td>
```

Comment:
@dvdksn I have 404 when using this href :grimacing: I guess this is because we have HTML in markdown so links are not processed as they should?